src/runtime/src/value.rs

```diff
@@ -395,3 +395,58 @@ pub fn value_size(value: &Value) -> usize {
         _ => 1,
     }
 }
+
+pub fn quick_sort(
```

Comment:
I didn't find a better place for this stuff. Still feels like it's a wrong place...

Reply:
It's OK, something like `value_sort.rs` could easily be extracted.